pa11y.js

```diff
@@ -0,0 +1,9 @@
+// This is our options configuration for Pa11y
+// https://github.com/pa11y/pa11y#configuration
+
+const options = {
+  timeout: 60000,
+  hideElements: '.skip-to, .is-visuallyhidden'
```

Comment:
@petronbot should we add `.visuallyhidden` here too, now that we decided to not deprecate it.